Source/Core/InputCommon/InputConfig.h

```diff
@@ -25,7 +25,14 @@ class InputConfig
 
   ~InputConfig();
 
-  bool LoadConfig(bool isGC);
+  enum class InputType
```

Comment:
I would name this InputClass, type should be reserved for other subdivisions, like the different gc controllers.